### DISCUSSION TITLE
orientation program

### DISCUSSION THREAD

**Discussion by shahshaik2000** [github/india#234] [Students]
how to access the orientation program?

**Comment by yashasvi28** [comment]
hey ! I'm not able to access the website it's crashing again and again

**Reply by shahshaik2000** [reply]
what should we do, we'll miss the program

**Reply by yashasvi28** [reply]
I've no idea

**Reply by yashasvi28** [reply]
https://github.com/github/india/discussions/125#discussioncomment-1433759
you can join from here you'll find a join here option

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
